tests/test_util.py

```diff
@@ -0,0 +1,20 @@
+from flow.util.misc import bidict
```

Comment:
Assuming this is fixed in #416 
```suggestion
from flow.util.misc import _bidict
```

Reply:
Yes, this is fixed in #416. My mistake.